package.json

```diff
@@ -93,6 +93,11 @@
                     "default": false,
                     "markdownDescription": "Equivalent to the startup option `--clean`."
                 },
+                "vscode-neovim.neovimCmd": {
+                    "type": "string",
```

Comment:
This should be array, so that the args can be passed without dealing with quoting.